testsuite/integration/manualmode/src/test/resources/arquillian.xml

```diff
@@ -52,7 +54,7 @@
                 <!-- AS7-4070 -->
                 <property name="waitForPorts">${as.debug.port:8787} ${as.managementPort:9990}</property>
                 <property name="waitForPortsTimeoutInSeconds">8</property>
-                <property name="modulePath">${basedir}/target/wildfly/modules</property>
```

Comment:
I'm not sure this should have been removed. It's an odd looking property to me, but it looks like it's been that way for a while so we should likely keep it.